README.md

```diff
@@ -160,142 +161,194 @@ Structured Extraction is guided by the provided schema. We support Pydantic Mode
 * [Page Classification](https://docs.tensorlake.ai/document-ingestion/parsing/page-classification)
 * [Signature Detection](https://docs.tensorlake.ai/document-ingestion/parsing/signature)
 
-## Data Workflows
+## Build Durable Agentic Applications in Python
+
+Tensorlake Agentic Applications allow you to chain Python functions into durable, distributed workflows. These functions can be deployed to a serverless backend with a single command.
```

Comment:
Tensorlake's Agent Runtime allows you to deploy agentic applications built in any framework on a distributed runtime which scales them as they get requests. The platform has built in durable execution to let applications restart from where they crash automatically.